src/draw/eve/lv_draw_eve.c

```diff
@@ -0,0 +1,169 @@
+/**
+ * @file lv_draw_eve.c
+ *
+ */
+
+/*  Created on: 3 dic 2023
+ *      Author: juanj
+ *
+ *  Modified by LVGL
+ */
+
+/*********************
+ *      INCLUDES
+ *********************/
+
+#include "lv_draw_eve_private.h"
+#if LV_USE_DRAW_EVE
+
+#include "../../core/lv_refr.h"
+#include "../../display/lv_display_private.h"
+#include "../../stdlib/lv_string.h"
+#include "lv_draw_eve_ram_g.h"
+#include "lv_draw_eve.h"
+#include "lv_eve.h"
+
+/*********************
+ *      DEFINES
+ *********************/
+
+#define DRAW_UNIT_ID_EVE 9
```

Comment:
General, non-blocking, question: Is there any criteria to select the ID of the draw unit?